src/main/java/magicbees/main/utils/compat/bloodmagic/ItemBloodBaseFrame.java

```diff
@@ -0,0 +1,139 @@
+        public float getMutationModifier(IBeeGenome genome, IBeeGenome mate, float currentModifier) {
+            return 1;
+        }
+
+        @Override
+        public float getLifespanModifier(IBeeGenome genome, IBeeGenome mate, float currentModifier) {
+            return 1;
+        }
+
+        @Override
+        public float getProductionModifier(IBeeGenome genome, float currentModifier) {
+            return 2;
+        }
+
+        @Override
+        public float getFloweringModifier(IBeeGenome genome, float currentModifier) {
+            return 1;
+        }
+
+        @Override
+        public float getGeneticDecay(IBeeGenome genome, float currentModifier) {
+            return (float) 0.8;
+        }
+
+        @Override
+        public boolean isSealed() {
+            return false;
+        }
+
+        @Override
+        public boolean isSelfLighted() {
+            return false;
+        }
+
+        @Override
+        public boolean isSunlightSimulated() {
+            return false;
+        }
+
+        @Override
+        public boolean isHellish() {
+            return false;
+        }
+
+        @Override
+        public float getTerritoryModifier(IBeeGenome genome, float currentModifier) {
+            return 1;
+        }
+    }
+
+    @Override
+    public void addInformation(ItemStack par1ItemStack, EntityPlayer par2EntityPlayer, List par3List, boolean par4) {
+        par3List.add(StatCollector.translateToLocal("You feel gross touching this"));
+        par3List.add(StatCollector.translateToLocal("Uses " + String.valueOf(energyUsed) + " LP per cycle"));
+
+        if (!(par1ItemStack.getTagCompound() == null)) {
+            par3List.add(
+                    StatCollector.translateToLocal("tooltip.owner.currentowner") + " "
+                            + par1ItemStack.getTagCompound().getString("ownerName"));
+        }
+
+    }
+
+    @Override
+    @SideOnly(Side.CLIENT)
+    public void registerIcons(IIconRegister iconRegister) {
+        this.itemIcon = iconRegister.registerIcon(CommonProxy.DOMAIN + ":bloodSoakedFrame");
+    }
+
+    @Override
+    public ItemStack onItemRightClick(ItemStack par1ItemStack, World par2World, EntityPlayer par3EntityPlayer) {
+
+        if (EnergyItems.checkAndSetItemOwner(par1ItemStack, par3EntityPlayer)) {
+            owner = par3EntityPlayer;
+            if (par1ItemStack.getItemDamage() > 0) {
+                if (EnergyItems.syphonBatteries(par1ItemStack, par3EntityPlayer, getEnergyUsed())) {
+                    par1ItemStack.setItemDamage(par1ItemStack.getItemDamage() - 1);
+                }
+            }
+        }
+
+        return par1ItemStack;
+    }
+
+    @Override
+    public ItemStack frameUsed(IBeeHousing housing, ItemStack frame, IBee queen, int wear) {
+        if (EnergyItems.canSyphonInContainer(frame, getEnergyUsed() * wear)) {
+            if (owner != null) {
+                EnergyItems.drainPlayerNetwork(owner, getEnergyUsed() * wear);
```

Comment:
That's intended

Reply:
@Alastors Imagine situation (with current implementation of these frames). I join server with many players and craft this frame. And at the moment I bind this frame to my network ***all*** such frames in all hives on server begin to drain LP from my network - and not from network of owner on tooltip.

Is ***that*** intended?

Reply:
Because my thought is that frame should drain LP from network of it's owner (like all BM tools do). And not from network of person who crafted and bound new frame.

Reply:
That's not what would happen

Reply:
And I'll prove it, just a moment

Reply:
<img width="523" alt="Screen Shot 2023-04-30 at 10 27 50 AM" src="https://user-images.githubusercontent.com/78517796/235358442-a3d648eb-8669-41c2-8ea3-3881580b9611.png">
<img width="534" alt="Screen Shot 2023-04-30 at 10 27 58 AM" src="https://user-images.githubusercontent.com/78517796/235358450-ec76da1b-4d60-4e59-b51f-623977ca21c7.png">
<img width="1440" alt="Screen Shot 2023-04-30 at 10 28 25 AM" src="https://user-images.githubusercontent.com/78517796/235358482-91f53f94-d511-4116-8aee-906e2e783f89.png">
<img width="1440" alt="Screen Shot 2023-04-30 at 10 28 34 AM" src="https://user-images.githubusercontent.com/78517796/235358489-c9856a95-cf18-4efa-a231-a9f45c6cfbf2.png">
<img width="397" alt="Screen Shot 2023-04-30 at 10 28 42 AM" src="https://user-images.githubusercontent.com/78517796/235358495-dff61a6d-4b7a-4747-bacb-7fee8e7d05d2.png">
@OneEyeMaker you were wrong, here's proof of such, if you were right this would've taken 2k out of my LP network, but no, it did not, it took 1k and the other frame dissolved like intended

Reply:
I can even film this working in action if you want me to